### ISSUE TITLE
Compiling error on debian jessie (raspberrypi2)

### ISSUE THREAD

**Issue by thundergreen** [majn/telegram-purple#244]
echo objs/telegram-purple.o && gcc -g -O2 -I/usr/include/glib-2.0 -I/usr/lib/arm-linux-gnueabihf/glib-2.0/include -I/usr/include/libpurple  -DHAVE_CONFIG_H -DENABLE_N
LS -DLOCALEDIR='"/usr/local/share/locale"' -Wall -Wextra -Wno-deprecated-declarations -Wno-unused-parameter -I. -I. -fno-strict-aliasing -fPIC  -I ./tgl -c -MP -MD -M
F dep/telegram-purple.d -MQ objs/telegram-purple.o -o objs/telegram-purple.o telegram-purple.c                                                                        
objs/telegram-purple.o                                                                                                                                                
telegram-purple.c: In function 'export_chat_link':                                                                                                                    
telegram-purple.c:338:22: error: 'TGLPF_ADMIN' undeclared (first use in this function)                                                                                
   if (! (P->flags & (TGLPF_ADMIN | TGLPF_CREATOR))) {                                                                                                                
                      ^                                                                                                                                               
telegram-purple.c:338:22: note: each undeclared identifier is reported only once for each function it appears in                                                      
telegram-purple.c:338:36: error: 'TGLPF_CREATOR' undeclared (first use in this function)                                                                              
   if (! (P->flags & (TGLPF_ADMIN | TGLPF_CREATOR))) {                                                                                                                
                                    ^                                                                                                                                 
telegram-purple.c: In function 'leave_and_delete_chat':                                                                                                               
telegram-purple.c:377:27: error: 'TGLCHF_LEFT' undeclared (first use in this function)                                                                                
     if (!(P->chat.flags & TGLCHF_LEFT)) {                                                                                                                            
                           ^                                                                                                                                          
telegram-purple.c: In function 'tgprpl_send_im':                                                                                                                      
telegram-purple.c:625:78: error: 'TGLCHF_CREATOR' undeclared (first use in this function)                                                                             
     if (tgl_get_peer_type (peer->id) == TGL_PEER_CHANNEL && ! (peer->flags & TGLCHF_CREATOR)) {                                                                      
                                                                              ^                                                                                       
Makefile:110: recipe for target 'objs/telegram-purple.o' failed                                                                                                       
make: *** [objs/telegram-purple.o] Error 1

Any idea how to fix this?

**Comment by majn** [comment]
Did you pull the updates from an older version? Seems like the libtgl submodule is out of date, maybe you forgot to do:

    git submodule update --recursive

**Comment by thundergreen** [comment]
@majn Seems to run now. I guess the debian repo is outa date and won't work. Cause i get internal server errors. That's why i want to build it from scratch

12273: 2016-02-23 17:31:41,868 INFO  libpurple: util:Reading file status.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                              
12273: 2016-02-23 17:31:41,868 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/status.xml does not exist (this is not necessarily an error)   
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate:CertificateVerifier x509, singleuse requested but not found.                                              
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate:CertificateVerifier singleuse registered                                                                  
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate:CertificatePool x509, ca requested but not found.                                                         
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate:CertificateScheme x509 requested but not found.                                                           
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate/x509/ca:Lazy init failed because an X.509 Scheme is not yet registered. Maybe it will be better later.    
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate/x509/ca:Init failed, probably because a dependency is not yet registered. It has been deferred to later.  
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate:CertificatePool ca registered                                                                             
12273: 2016-02-23 17:31:41,869 INFO  libpurple: certificate:CertificatePool x509, tls_peers requested but not found.                                                  
12273: 2016-02-23 17:31:41,870 INFO  libpurple: certificate:CertificatePool tls_peers registered                                                                      
12273: 2016-02-23 17:31:41,870 INFO  libpurple: certificate:CertificateVerifier x509, tls_cached requested but not found.                                             
12273: 2016-02-23 17:31:41,871 INFO  libpurple: certificate:CertificateVerifier tls_cached registered                                                                 
12273: 2016-02-23 17:31:41,871 INFO  libpurple: prefs:/purple/logging/format changed, scheduling save.                                                                
12273: 2016-02-23 17:31:41,871 INFO  libpurple: prefs:/purple/logging/format changed, scheduling save.                                                                
12273: 2016-02-23 17:31:41,885 INFO  libpurple: prefs:/purple/proxy/type changed, scheduling save.                                                                    
12273: 2016-02-23 17:31:41,885 INFO  libpurple: prefs:/purple/proxy/host changed, scheduling save.                                                                    
12273: 2016-02-23 17:31:41,885 INFO  libpurple: prefs:/purple/proxy/port changed, scheduling save.                                                                    
12273: 2016-02-23 17:31:41,885 INFO  libpurple: prefs:/purple/proxy/username changed, scheduling save.                                                                
12273: 2016-02-23 17:31:41,885 INFO  libpurple: prefs:/purple/proxy/password changed, scheduling save.                                                                
12273: 2016-02-23 17:31:41,902 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256: Enabled                                                         
12273: 2016-02-23 17:31:41,902 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256: Enabled                                                           
12273: 2016-02-23 17:31:41,902 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA: Enabled                                                            
12273: 2016-02-23 17:31:41,902 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA: Enabled                                                            
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA: Enabled                                                              
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256: Disabled                                                        
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256: Enabled                                                           
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA: Enabled                                                              
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                          
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                            
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_RC4_128_SHA: Enabled                                                                
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_RC4_128_SHA: Enabled                                                                  
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_128_GCM_SHA256: Enabled                                                             
12273: 2016-02-23 17:31:41,903 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_128_CBC_SHA: Enabled                                                                
12273: 2016-02-23 17:31:41,904 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_AES_128_CBC_SHA: Enabled                                                                
12273: 2016-02-23 17:31:41,904 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_128_CBC_SHA256: Enabled                                                             
12273: 2016-02-23 17:31:41,904 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_CAMELLIA_128_CBC_SHA: Disabled                                                          
12273: 2016-02-23 17:31:41,904 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_CAMELLIA_128_CBC_SHA: Disabled                                                          
12273: 2016-02-23 17:31:41,904 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_256_CBC_SHA: Enabled                                                                
12273: 2016-02-23 17:31:41,904 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_AES_256_CBC_SHA: Disabled                                                               
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_256_CBC_SHA256: Enabled                                                             
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_CAMELLIA_256_CBC_SHA: Disabled                                                          
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_CAMELLIA_256_CBC_SHA: Disabled                                                          
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                              
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_3DES_EDE_CBC_SHA: Disabled                                                              
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_RC4_128_SHA: Disabled                                                                   
12273: 2016-02-23 17:31:41,905 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA: Disabled                                                            
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_AES_128_CBC_SHA: Disabled                                                              
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA: Disabled                                                            
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_AES_256_CBC_SHA: Disabled                                                              
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                           
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                             
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_RC4_128_SHA: Disabled                                                                
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_RC4_128_SHA: Disabled                                                                  
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_128_GCM_SHA256: Disabled                                                                
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_128_CBC_SHA: Enabled                                                                    
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_128_CBC_SHA256: Disabled                                                                
12273: 2016-02-23 17:31:41,906 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_CAMELLIA_128_CBC_SHA: Disabled                                                              
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_256_CBC_SHA: Enabled                                                                    
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_256_CBC_SHA256: Disabled                                                                
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_CAMELLIA_256_CBC_SHA: Disabled                                                              
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_SEED_CBC_SHA: Disabled                                                                      
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - SSL_RSA_FIPS_WITH_3DES_EDE_CBC_SHA: Disabled                                                             
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                                  
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_RC4_128_SHA: Disabled                                                                       
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_RC4_128_MD5: Disabled                                                                       
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_DES_CBC_SHA: Disabled                                                                   
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_DES_CBC_SHA: Disabled                                                                   
12273: 2016-02-23 17:31:41,907 INFO  libpurple: nss:Cipher - SSL_RSA_FIPS_WITH_DES_CBC_SHA: Disabled                                                                  
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_DES_CBC_SHA: Disabled                                                                       
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT1024_WITH_RC4_56_SHA: Disabled                                                             
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT1024_WITH_DES_CBC_SHA: Disabled                                                            
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT_WITH_RC4_40_MD5: Disabled                                                                 
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT_WITH_RC2_CBC_40_MD5: Disabled                                                             
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_NULL_SHA: Disabled                                                                  
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_NULL_SHA: Disabled                                                                    
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_NULL_SHA: Disabled                                                                     
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_NULL_SHA: Disabled                                                                   
12273: 2016-02-23 17:31:41,908 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_NULL_SHA: Disabled                                                                          
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_NULL_SHA256: Disabled                                                                       
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_NULL_MD5: Disabled                                                                          
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - SSL_CK_RC4_128_WITH_MD5: Disabled                                                                        
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - SSL_CK_RC2_128_CBC_WITH_MD5: Disabled                                                                    
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - SSL_CK_DES_192_EDE3_CBC_WITH_MD5: Disabled                                                               
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - SSL_CK_DES_64_CBC_WITH_MD5: Disabled                                                                     
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - SSL_CK_RC4_128_EXPORT40_WITH_MD5: Disabled                                                               
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:Cipher - SSL_CK_RC2_128_CBC_EXPORT40_WITH_MD5: Disabled                                                           
12273: 2016-02-23 17:31:41,909 INFO  libpurple: nss:TLS supported versions: 0x0300 through 0x0303                                                                     
12273: 2016-02-23 17:31:41,910 INFO  libpurple: nss:TLS versions allowed by default: 0x0300 through 0x0301                                                            
12273: 2016-02-23 17:31:41,910 INFO  libpurple: nss:Changed allowed TLS versions to 0x0300 through 0x0303                                                             
12273: 2016-02-23 17:31:41,910 INFO  libpurple: certificate:CertificateScheme x509 requested but not found.                                                           
12273: 2016-02-23 17:31:41,910 INFO  libpurple: certificate:CertificateScheme x509 registered                                                                         
12273: 2016-02-23 17:31:41,910 INFO  libpurple: util:Reading file smileys.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                             
12273: 2016-02-23 17:31:41,910 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/smileys.xml does not exist (this is not necessarily an error)  
12273: 2016-02-23 17:31:41,911 INFO  libpurple: stun:using server                                                                                                     
12273: 2016-02-23 17:31:41,911 INFO  libpurple: util:Reading file blist.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                               
12273: 2016-02-23 17:31:41,912 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/blist.xml does not exist (this is not necessarily an error)    
12273: 2016-02-23 17:31:41,912 INFO  libpurple: prefs:Reading /var/lib/spectrum2/telegram.emevth.no-ip.biz/prefs.xml                                                  
12273: 2016-02-23 17:31:41,913 INFO  libpurple: prefs:/plugins/core/nss_prefs/cipher_list changed, scheduling save.                                                   
12273: 2016-02-23 17:31:41,915 INFO  libpurple: prefs:Finished reading /var/lib/spectrum2/telegram.emevth.no-ip.biz/prefs.xml                                         
12273: 2016-02-23 17:31:41,922 INFO  backend: Creating account with name '+33781852856' and protocol 'prpl-telegram'                                                  
12273: 2016-02-23 17:31:41,923 INFO  libpurple: prpl-telegram:tgprpl_status_types()                                                                                   
12273: 2016-02-23 17:31:41,924 INFO  libpurple: account:Connecting to account +33781852856.                                                                           
12273: 2016-02-23 17:31:41,924 INFO  libpurple: connection:Connecting. gc = 0xda6ac0                                                                                  
12273: 2016-02-23 17:31:41,924 INFO  libpurple: prpl-telegram:tgprpl_login(): Purple is telling the prpl to connect the account                                       
12273: 2016-02-23 17:31:41,925 INFO  libpurple: prpl-telegram:base configuration path: '/var/lib/spectrum2/telegram.emevth.no-ip.biz/telegram-purple/+33781852856'    
12273: 2016-02-23 17:31:41,925 INFO  libpurple: prpl-telegram:trying global pubkey at /etc/telegram-purple/server.tglpub                                              
12273: 2016-02-23 17:31:41,925 INFO  libpurple: prpl-telegram:read pubkey file: n_len=256 e=65537                                                                     
12273: 2016-02-23 17:31:41,925 INFO  libpurple: prpl-telegram:using global pubkey                                                                                     
12273: 2016-02-23 17:31:41,933 INFO  libpurple: prpl-telegram:'direct' public key loaded successfully                                                                 
12273: 2016-02-23 17:31:41,933 INFO  libpurple: prpl-telegram:initializing empty auth file                                                                            
12273: 2016-02-23 17:31:41,933 INFO  libpurple: prpl-telegram:DC1 '' update: 149.154.175.50:443                                                                       
12273: 2016-02-23 17:31:41,934 INFO  libpurple: prpl-telegram:DC2 '' update: 149.154.167.51:443                                                                       
12273: 2016-02-23 17:31:41,935 INFO  libpurple: prpl-telegram:DC3 '' update: 149.154.175.100:443                                                                      
12273: 2016-02-23 17:31:41,935 INFO  libpurple: prpl-telegram:DC4 '' update: 149.154.167.91:443                                                                       
12273: 2016-02-23 17:31:41,936 INFO  libpurple: prpl-telegram:DC5 '' update: 149.154.171.5:443                                                                        
12273: 2016-02-23 17:31:42,039 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.175.50                                                                     
12273: 2016-02-23 17:31:42,040 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.167.51                                                                     
12273: 2016-02-23 17:31:42,040 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.175.100                                                                    
12273: 2016-02-23 17:31:42,040 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.167.91                                                                     
12273: 2016-02-23 17:31:42,040 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.171.5                                                                      
12273: 2016-02-23 17:31:42,041 INFO  libpurple: dnsquery:IP resolved for 149.154.175.50                                                                               
12273: 2016-02-23 17:31:42,041 INFO  libpurple: proxy:Attempting connection to 149.154.175.50                                                                         
12273: 2016-02-23 17:31:42,041 INFO  libpurple: proxy:Connecting to 149.154.175.50:443 with no proxy                                                                  
12273: 2016-02-23 17:31:42,041 INFO  libpurple: proxy:Connection in progress                                                                                          
12273: 2016-02-23 17:31:42,041 INFO  libpurple: dnsquery:IP resolved for 149.154.167.51                                                                               
12273: 2016-02-23 17:31:42,042 INFO  libpurple: proxy:Attempting connection to 149.154.167.51                                                                         
12273: 2016-02-23 17:31:42,042 INFO  libpurple: proxy:Connecting to 149.154.167.51:443 with no proxy                                                                  
12273: 2016-02-23 17:31:42,042 INFO  libpurple: proxy:Connection in progress                                                                                          
12273: 2016-02-23 17:31:42,042 INFO  libpurple: dnsquery:IP resolved for 149.154.175.100                                                                              
12273: 2016-02-23 17:31:42,042 INFO  libpurple: proxy:Attempting connection to 149.154.175.100                                                                        
12273: 2016-02-23 17:31:42,043 INFO  libpurple: proxy:Connecting to 149.154.175.100:443 with no proxy                                                                 
12273: 2016-02-23 17:31:42,043 INFO  libpurple: proxy:Connection in progress                                                                                          
12273: 2016-02-23 17:31:42,043 INFO  libpurple: dnsquery:IP resolved for 149.154.167.91                                                                               
12273: 2016-02-23 17:31:42,043 INFO  libpurple: proxy:Attempting connection to 149.154.167.91                                                                         
12273: 2016-02-23 17:31:42,043 INFO  libpurple: proxy:Connecting to 149.154.167.91:443 with no proxy                                                                  
12273: 2016-02-23 17:31:42,043 INFO  libpurple: proxy:Connection in progress                                                                                          
12273: 2016-02-23 17:31:42,044 INFO  libpurple: dnsquery:IP resolved for 149.154.171.5                                                                                
12273: 2016-02-23 17:31:42,044 INFO  libpurple: proxy:Attempting connection to 149.154.171.5                                                                          
12273: 2016-02-23 17:31:42,044 INFO  libpurple: proxy:Connecting to 149.154.171.5:443 with no proxy                                                                   
12273: 2016-02-23 17:31:42,044 INFO  libpurple: proxy:Connection in progress                                                                                          
12273: 2016-02-23 17:31:42,086 INFO  libpurple: proxy:Connecting to 149.154.167.51:443.                                                                               
12273: 2016-02-23 17:31:42,086 INFO  libpurple: proxy:Connected to 149.154.167.51:443.                                                                                
12273: 2016-02-23 17:31:42,089 INFO  libpurple: proxy:Connecting to 149.154.167.91:443.                                                                               
12273: 2016-02-23 17:31:42,090 INFO  libpurple: proxy:Connected to 149.154.167.91:443.                                                                                
12273: 2016-02-23 17:31:42,092 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #2 becomed ready                                                        
12273: 2016-02-23 17:31:42,094 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #4 becomed ready                                                        
12273: 2016-02-23 17:31:42,855 ERROR backend: g_log purple_account_set_public_alias: assertion 'purple_account_is_connected(account)' failed                          
12273: 2016-02-23 17:31:42,856 INFO  libpurple: util:Writing file /var/lib/spectrum2/telegram.emevth.no-ip.biz/icons/935c8cc52a717d70bf38fb80229f8c3ad2b89674.jpg     
12273: 2016-02-23 17:31:43,557 INFO  libpurple: proxy:Connecting to 149.154.175.50:443.                                                                               
12273: 2016-02-23 17:31:43,558 INFO  libpurple: proxy:Connected to 149.154.175.50:443.                                                                                
12273: 2016-02-23 17:31:43,561 INFO  libpurple: proxy:Connecting to 149.154.175.100:443.                                                                              
12273: 2016-02-23 17:31:43,561 INFO  libpurple: proxy:Connected to 149.154.175.100:443.                                                                               
12273: 2016-02-23 17:31:43,563 INFO  libpurple: proxy:Connecting to 149.154.171.5:443.                                                                                
12273: 2016-02-23 17:31:43,564 INFO  libpurple: proxy:Connected to 149.154.171.5:443.                                                                                 
12273: 2016-02-23 17:31:43,679 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #1 becomed ready                                                        
12273: 2016-02-23 17:31:43,680 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #3 becomed ready                                                        
12273: 2016-02-23 17:31:43,681 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #5 becomed ready                                                        
12307: 2016-02-23 17:32:51,718 INFO  backend: Setting libpurple plugins directory to: ./plugins                                                                       
12307: 2016-02-23 17:32:51,719 INFO  backend: Setting libpurple cacerts directory to: ./ca-certs                                                                      
12307: 2016-02-23 17:32:51,719 INFO  backend: Setting libpurple user directory to: /var/lib/spectrum2/telegram.emevth.no-ip.biz                                       
12307: 2016-02-23 17:32:52,329 INFO  backend: Will use glib based event loop                                                                                          
12307: 2016-02-23 17:32:52,330 INFO  libpurple: prefs:Reading /var/lib/spectrum2/telegram.emevth.no-ip.biz/prefs.xml                                                  
12307: 2016-02-23 17:32:52,334 INFO  libpurple: prefs:Finished reading /var/lib/spectrum2/telegram.emevth.no-ip.biz/prefs.xml                                         
12307: 2016-02-23 17:32:52,334 INFO  libpurple: dbus:Failed to get connection: Unable to autolaunch a dbus-daemon without a $DISPLAY for X11                          
12307: 2016-02-23 17:32:52,335 INFO  libpurple: plugins:probing /usr/lib/purple-2/libzephyr.so                                                                        
12307: 2016-02-23 17:32:52,341 INFO  libpurple: plugins:probing /usr/lib/purple-2/libmsn.so                                                                           
12307: 2016-02-23 17:32:52,348 INFO  libpurple: plugins:probing /usr/lib/purple-2/liboscar.so                                                                         
12307: 2016-02-23 17:32:52,353 INFO  libpurple: plugins:/usr/lib/purple-2/liboscar.so is not usable because the 'purple_init_plugin' symbol could not be found.  Does 
the plugin call the PURPLE_INIT_PLUGIN() macro?                                                                                                                       
12307: 2016-02-23 17:32:52,354 INFO  libpurple: plugins:probing /usr/lib/purple-2/libaim.so                                                                           
12307: 2016-02-23 17:32:52,361 INFO  libpurple: plugins:probing /usr/lib/purple-2/dbus-example.so                                                                     
12307: 2016-02-23 17:32:52,363 INFO  libpurple: plugins:probing /usr/lib/purple-2/libwhatsapp.so                                                                      
12307: 2016-02-23 17:32:52,467 INFO  libpurple: plugins:probing /usr/lib/purple-2/joinpart.so                                                                         
12307: 2016-02-23 17:32:52,469 INFO  libpurple: plugins:probing /usr/lib/purple-2/tcl.so                                                                              
12307: 2016-02-23 17:32:52,520 INFO  libpurple: plugins:probing /usr/lib/purple-2/libxmpp.so                                                                          
12307: 2016-02-23 17:32:52,529 INFO  libpurple: plugins:probing /usr/lib/purple-2/libskypeweb.so                                                                      
12307: 2016-02-23 17:32:52,534 INFO  libpurple: plugins:probing /usr/lib/purple-2/perl.so                                                                             
12307: 2016-02-23 17:32:52,552 INFO  libpurple: plugins:probing /usr/lib/purple-2/libyahoo.so                                                                         
12307: 2016-02-23 17:32:52,557 INFO  libpurple: plugins:probing /usr/lib/purple-2/libbonjour.so                                                                       
12307: 2016-02-23 17:32:52,564 INFO  libpurple: plugins:probing /usr/lib/purple-2/libirc.so                                                                           
12307: 2016-02-23 17:32:52,567 INFO  libpurple: plugins:probing /usr/lib/purple-2/libymsg.so                                                                          
12307: 2016-02-23 17:32:52,569 INFO  libpurple: plugins:/usr/lib/purple-2/libymsg.so is not usable because the 'purple_init_plugin' symbol could not be found.  Does t
he plugin call the PURPLE_INIT_PLUGIN() macro?                                                                                                                        
12307: 2016-02-23 17:32:52,569 INFO  libpurple: plugins:probing /usr/lib/purple-2/log_reader.so                                                                       
12307: 2016-02-23 17:32:52,571 INFO  libpurple: plugins:probing /usr/lib/purple-2/libsimple.so                                                                        
12307: 2016-02-23 17:32:52,573 INFO  libpurple: plugins:probing /usr/lib/purple-2/libnovell.so                                                                        
12307: 2016-02-23 17:32:52,577 INFO  libpurple: plugins:probing /usr/lib/purple-2/libyahoojp.so                                                                       
12307: 2016-02-23 17:32:52,579 INFO  libpurple: plugins:probing /usr/lib/purple-2/libicq.so                                                                           
12307: 2016-02-23 17:32:52,581 INFO  libpurple: plugins:probing /usr/lib/purple-2/nss-prefs.so                                                                        
12307: 2016-02-23 17:32:52,608 INFO  libpurple: plugins:probing /usr/lib/purple-2/libsametime.so                                                                      
12307: 2016-02-23 17:32:52,614 INFO  libpurple: plugins:/usr/lib/purple-2/libsametime.so has a prefs_info, but is a prpl. This is no longer supported.                
12307: 2016-02-23 17:32:52,615 INFO  libpurple: plugins:probing /usr/lib/purple-2/libmxit.so                                                                          
12307: 2016-02-23 17:32:52,618 INFO  libpurple: prpl-loubserp-mxit:Loading MXit libPurple plugin...                                                                   
12307: 2016-02-23 17:32:52,619 INFO  libpurple: plugins:probing /usr/lib/purple-2/ssl.so                                                                              
12307: 2016-02-23 17:32:52,620 INFO  libpurple: plugins:probing /usr/lib/purple-2/autoaccept.so                                                                       
12307: 2016-02-23 17:32:52,622 INFO  libpurple: plugins:probing /usr/lib/purple-2/telegram-purple.so                                                                  
12307: 2016-02-23 17:32:52,653 INFO  libpurple: plugins:probing /usr/lib/purple-2/libmyspace.so                                                                       
12307: 2016-02-23 17:32:52,656 INFO  libpurple: plugins:probing /usr/lib/purple-2/libjabber.so                                                                        
12307: 2016-02-23 17:32:52,657 INFO  libpurple: plugins:/usr/lib/purple-2/libjabber.so is not usable because the 'purple_init_plugin' symbol could not be found.  Does
 the plugin call the PURPLE_INIT_PLUGIN() macro?                                                                                                                      
12307: 2016-02-23 17:32:52,658 INFO  libpurple: plugins:probing /usr/lib/purple-2/offlinemsg.so                                                                       
12307: 2016-02-23 17:32:52,659 INFO  libpurple: plugins:probing /usr/lib/purple-2/ssl-nss.so                                                                          
12307: 2016-02-23 17:32:52,663 INFO  libpurple: plugins:probing /usr/lib/purple-2/newline.so                                                                          
12307: 2016-02-23 17:32:52,664 INFO  libpurple: plugins:probing /usr/lib/purple-2/statenotify.so                                                                      
12307: 2016-02-23 17:32:52,666 INFO  libpurple: plugins:probing /usr/lib/purple-2/libfacebook.so                                                                      
12307: 2016-02-23 17:32:52,671 INFO  libpurple: plugins:probing /usr/lib/purple-2/libgg.so                                                                            
12307: 2016-02-23 17:32:52,674 INFO  libpurple: plugins:probing /usr/lib/purple-2/buddynote.so                                                                        
12307: 2016-02-23 17:32:52,675 INFO  libpurple: plugins:probing /usr/lib/purple-2/idle.so                                                                             
12307: 2016-02-23 17:32:52,677 INFO  libpurple: plugins:probing /usr/lib/purple-2/psychic.so                                                                          
12307: 2016-02-23 17:32:52,732 INFO  libpurple: util:Reading file xmpp-caps.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                           
12307: 2016-02-23 17:32:52,732 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/xmpp-caps.xml does not exist (this is not necessarily an error)
12307: 2016-02-23 17:32:52,733 INFO  libpurple: jabber:creating hash tables for data objects                                                                          
12307: 2016-02-23 17:32:52,735 INFO  libpurple: gg:Loading Gadu-Gadu protocol plugin with libgadu 1.12.0...                                                           
12307: 2016-02-23 17:32:52,735 INFO  libpurple: prefs:/purple/status/scores/offline changed, scheduling save.                                                         
12307: 2016-02-23 17:32:52,735 INFO  libpurple: prefs:/purple/status/scores/available changed, scheduling save.                                                       
12307: 2016-02-23 17:32:52,735 INFO  libpurple: prefs:/purple/status/scores/invisible changed, scheduling save.                                                       
12307: 2016-02-23 17:32:52,736 INFO  libpurple: prefs:/purple/status/scores/away changed, scheduling save.                                                            
12307: 2016-02-23 17:32:52,736 INFO  libpurple: prefs:/purple/status/scores/extended_away changed, scheduling save.                                                   
12307: 2016-02-23 17:32:52,736 INFO  libpurple: prefs:/purple/status/scores/idle changed, scheduling save.                                                            
12307: 2016-02-23 17:32:52,736 INFO  libpurple: prefs:/purple/status/scores/offline_msg changed, scheduling save.                                                     
12307: 2016-02-23 17:32:52,736 INFO  libpurple: util:Reading file accounts.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                            
12307: 2016-02-23 17:32:52,736 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/accounts.xml does not exist (this is not necessarily an error) 
12307: 2016-02-23 17:32:52,737 INFO  libpurple: util:Reading file status.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                              
12307: 2016-02-23 17:32:52,737 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/status.xml does not exist (this is not necessarily an error)   
12307: 2016-02-23 17:32:52,737 INFO  libpurple: certificate:CertificateVerifier x509, singleuse requested but not found.                                              
12307: 2016-02-23 17:32:52,737 INFO  libpurple: certificate:CertificateVerifier singleuse registered                                                                  
12307: 2016-02-23 17:32:52,737 INFO  libpurple: certificate:CertificatePool x509, ca requested but not found.                                                         
12307: 2016-02-23 17:32:52,737 INFO  libpurple: certificate:CertificateScheme x509 requested but not found.                                                           
12307: 2016-02-23 17:32:52,738 INFO  libpurple: certificate/x509/ca:Lazy init failed because an X.509 Scheme is not yet registered. Maybe it will be better later.    
12307: 2016-02-23 17:32:52,738 INFO  libpurple: certificate/x509/ca:Init failed, probably because a dependency is not yet registered. It has been deferred to later.  
12307: 2016-02-23 17:32:52,738 INFO  libpurple: certificate:CertificatePool ca registered                                                                             
12307: 2016-02-23 17:32:52,738 INFO  libpurple: certificate:CertificatePool x509, tls_peers requested but not found.                                                  
12307: 2016-02-23 17:32:52,738 INFO  libpurple: certificate:CertificatePool tls_peers registered                                                                      
12307: 2016-02-23 17:32:52,738 INFO  libpurple: certificate:CertificateVerifier x509, tls_cached requested but not found.                                             
12307: 2016-02-23 17:32:52,739 INFO  libpurple: certificate:CertificateVerifier tls_cached registered                                                                 
12307: 2016-02-23 17:32:52,739 INFO  libpurple: prefs:/purple/logging/format changed, scheduling save.                                                                
12307: 2016-02-23 17:32:52,739 INFO  libpurple: prefs:/purple/logging/format changed, scheduling save.                                                                
12307: 2016-02-23 17:32:52,746 INFO  libpurple: prefs:/purple/proxy/type changed, scheduling save.                                                                    
12307: 2016-02-23 17:32:52,746 INFO  libpurple: prefs:/purple/proxy/host changed, scheduling save.                                                                    
12307: 2016-02-23 17:32:52,747 INFO  libpurple: prefs:/purple/proxy/port changed, scheduling save.                                                                    
12307: 2016-02-23 17:32:52,747 INFO  libpurple: prefs:/purple/proxy/username changed, scheduling save.                                                                
12307: 2016-02-23 17:32:52,747 INFO  libpurple: prefs:/purple/proxy/password changed, scheduling save.                                                                
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256: Enabled                                                         
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256: Enabled                                                           
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA: Enabled                                                            
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA: Enabled                                                            
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA: Enabled                                                              
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256: Disabled                                                        
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256: Enabled                                                           
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA: Enabled                                                              
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                          
12307: 2016-02-23 17:32:52,762 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                            
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_RC4_128_SHA: Enabled                                                                
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_RC4_128_SHA: Enabled                                                                  
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_128_GCM_SHA256: Enabled                                                             
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_128_CBC_SHA: Enabled                                                                
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_AES_128_CBC_SHA: Enabled                                                                
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_128_CBC_SHA256: Enabled                                                             
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_CAMELLIA_128_CBC_SHA: Disabled                                                          
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_CAMELLIA_128_CBC_SHA: Disabled                                                          
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_256_CBC_SHA: Enabled                                                                
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_AES_256_CBC_SHA: Disabled                                                               
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_AES_256_CBC_SHA256: Enabled                                                             
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_CAMELLIA_256_CBC_SHA: Disabled                                                          
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_CAMELLIA_256_CBC_SHA: Disabled                                                          
12307: 2016-02-23 17:32:52,763 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                              
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_3DES_EDE_CBC_SHA: Disabled                                                              
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_RC4_128_SHA: Disabled                                                                   
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA: Disabled                                                            
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_AES_128_CBC_SHA: Disabled                                                              
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA: Disabled                                                            
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_AES_256_CBC_SHA: Disabled                                                              
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                           
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                             
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_RC4_128_SHA: Disabled                                                                
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_RC4_128_SHA: Disabled                                                                  
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_128_GCM_SHA256: Disabled                                                                
12307: 2016-02-23 17:32:52,764 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_128_CBC_SHA: Enabled                                                                    
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_128_CBC_SHA256: Disabled                                                                
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_CAMELLIA_128_CBC_SHA: Disabled                                                              
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_256_CBC_SHA: Enabled                                                                    
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_AES_256_CBC_SHA256: Disabled                                                                
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_CAMELLIA_256_CBC_SHA: Disabled                                                              
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_SEED_CBC_SHA: Disabled                                                                      
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - SSL_RSA_FIPS_WITH_3DES_EDE_CBC_SHA: Disabled                                                             
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_3DES_EDE_CBC_SHA: Disabled                                                                  
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_RC4_128_SHA: Disabled                                                                       
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_RC4_128_MD5: Disabled                                                                       
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_DHE_RSA_WITH_DES_CBC_SHA: Disabled                                                                   
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_DHE_DSS_WITH_DES_CBC_SHA: Disabled                                                                   
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - SSL_RSA_FIPS_WITH_DES_CBC_SHA: Disabled                                                                  
12307: 2016-02-23 17:32:52,765 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_DES_CBC_SHA: Disabled                                                                       
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT1024_WITH_RC4_56_SHA: Disabled                                                             
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT1024_WITH_DES_CBC_SHA: Disabled                                                            
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT_WITH_RC4_40_MD5: Disabled                                                                 
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_EXPORT_WITH_RC2_CBC_40_MD5: Disabled                                                             
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_ECDHE_ECDSA_WITH_NULL_SHA: Disabled                                                                  
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_ECDHE_RSA_WITH_NULL_SHA: Disabled                                                                    
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_ECDH_RSA_WITH_NULL_SHA: Disabled                                                                     
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_ECDH_ECDSA_WITH_NULL_SHA: Disabled                                                                   
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_NULL_SHA: Disabled                                                                          
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_NULL_SHA256: Disabled                                                                       
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - TLS_RSA_WITH_NULL_MD5: Disabled                                                                          
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - SSL_CK_RC4_128_WITH_MD5: Disabled                                                                        
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - SSL_CK_RC2_128_CBC_WITH_MD5: Disabled                                                                    
12307: 2016-02-23 17:32:52,766 INFO  libpurple: nss:Cipher - SSL_CK_DES_192_EDE3_CBC_WITH_MD5: Disabled                                                               
12307: 2016-02-23 17:32:52,767 INFO  libpurple: nss:Cipher - SSL_CK_DES_64_CBC_WITH_MD5: Disabled                                                                     
12307: 2016-02-23 17:32:52,767 INFO  libpurple: nss:Cipher - SSL_CK_RC4_128_EXPORT40_WITH_MD5: Disabled                                                               
12307: 2016-02-23 17:32:52,767 INFO  libpurple: nss:Cipher - SSL_CK_RC2_128_CBC_EXPORT40_WITH_MD5: Disabled                                                           
12307: 2016-02-23 17:32:52,767 INFO  libpurple: nss:TLS supported versions: 0x0300 through 0x0303                                                                     
12307: 2016-02-23 17:32:52,767 INFO  libpurple: nss:TLS versions allowed by default: 0x0300 through 0x0301                                                            
12307: 2016-02-23 17:32:52,767 INFO  libpurple: nss:Changed allowed TLS versions to 0x0300 through 0x0303                                                             
12307: 2016-02-23 17:32:52,767 INFO  libpurple: certificate:CertificateScheme x509 requested but not found.                                                           
12307: 2016-02-23 17:32:52,767 INFO  libpurple: certificate:CertificateScheme x509 registered                                                                         
12307: 2016-02-23 17:32:52,767 INFO  libpurple: util:Reading file smileys.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                             
12307: 2016-02-23 17:32:52,768 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/smileys.xml does not exist (this is not necessarily an error)  
12307: 2016-02-23 17:32:52,768 INFO  libpurple: stun:using server                                                                                                     
12307: 2016-02-23 17:32:52,768 INFO  libpurple: util:Reading file blist.xml from directory /var/lib/spectrum2/telegram.emevth.no-ip.biz                               
12307: 2016-02-23 17:32:52,768 INFO  libpurple: util:File /var/lib/spectrum2/telegram.emevth.no-ip.biz/blist.xml does not exist (this is not necessarily an error)    
12307: 2016-02-23 17:32:52,768 INFO  libpurple: prefs:Reading /var/lib/spectrum2/telegram.emevth.no-ip.biz/prefs.xml                                                  
12307: 2016-02-23 17:32:52,770 INFO  libpurple: prefs:/plugins/core/nss_prefs/cipher_list changed, scheduling save.                                                   
12307: 2016-02-23 17:32:52,771 INFO  libpurple: prefs:Finished reading /var/lib/spectrum2/telegram.emevth.no-ip.biz/prefs.xml                                         
12307: 2016-02-23 17:32:52,778 INFO  backend: Creating account with name '+33781852856' and protocol 'prpl-telegram'                                                  
12307: 2016-02-23 17:32:52,778 INFO  libpurple: prpl-telegram:tgprpl_status_types()                                                                                   
12307: 2016-02-23 17:32:52,779 INFO  libpurple: account:Connecting to account +33781852856.                                                                           
12307: 2016-02-23 17:32:52,779 INFO  libpurple: connection:Connecting. gc = 0x583b78                                                                                  
12307: 2016-02-23 17:32:52,779 INFO  libpurple: prpl-telegram:tgprpl_login(): Purple is telling the prpl to connect the account                                       
12307: 2016-02-23 17:32:52,780 INFO  libpurple: prpl-telegram:base configuration path: '/var/lib/spectrum2/telegram.emevth.no-ip.biz/telegram-purple/+33781852856'    
12307: 2016-02-23 17:32:52,780 INFO  libpurple: prpl-telegram:trying global pubkey at /etc/telegram-purple/server.tglpub                                              
12307: 2016-02-23 17:32:52,780 INFO  libpurple: prpl-telegram:read pubkey file: n_len=256 e=65537                                                                     
12307: 2016-02-23 17:32:52,780 INFO  libpurple: prpl-telegram:using global pubkey                                                                                     
12307: 2016-02-23 17:32:52,781 INFO  libpurple: prpl-telegram:'direct' public key loaded successfully                                                                 
12307: 2016-02-23 17:32:52,782 INFO  libpurple: prpl-telegram:initializing empty auth file                                                                            
12307: 2016-02-23 17:32:52,782 INFO  libpurple: prpl-telegram:DC1 '' update: 149.154.175.50:443                                                                       
12307: 2016-02-23 17:32:52,782 INFO  libpurple: prpl-telegram:DC2 '' update: 149.154.167.51:443                                                                       
12307: 2016-02-23 17:32:52,782 INFO  libpurple: prpl-telegram:DC3 '' update: 149.154.175.100:443                                                                      
12307: 2016-02-23 17:32:52,782 INFO  libpurple: prpl-telegram:DC4 '' update: 149.154.167.91:443                                                                       
12307: 2016-02-23 17:32:52,782 INFO  libpurple: prpl-telegram:DC5 '' update: 149.154.171.5:443                                                                        
12307: 2016-02-23 17:32:52,885 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.175.50                                                                     
12307: 2016-02-23 17:32:52,886 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.167.51                                                                     
12307: 2016-02-23 17:32:52,886 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.175.100                                                                    
12307: 2016-02-23 17:32:52,886 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.167.91                                                                     
12307: 2016-02-23 17:32:52,886 INFO  libpurple: dnsquery:Performing DNS lookup for 149.154.171.5                                                                      
12307: 2016-02-23 17:32:52,886 INFO  libpurple: dnsquery:IP resolved for 149.154.175.50                                                                               
12307: 2016-02-23 17:32:52,886 INFO  libpurple: proxy:Attempting connection to 149.154.175.50                                                                         
12307: 2016-02-23 17:32:52,887 INFO  libpurple: proxy:Connecting to 149.154.175.50:443 with no proxy                                                                  
12307: 2016-02-23 17:32:52,887 INFO  libpurple: proxy:Connection in progress                                                                                          
12307: 2016-02-23 17:32:52,887 INFO  libpurple: dnsquery:IP resolved for 149.154.167.51                                                                               
12307: 2016-02-23 17:32:52,887 INFO  libpurple: proxy:Attempting connection to 149.154.167.51                                                                         
12307: 2016-02-23 17:32:52,887 INFO  libpurple: proxy:Connecting to 149.154.167.51:443 with no proxy                                                                  
12307: 2016-02-23 17:32:52,887 INFO  libpurple: proxy:Connection in progress                                                                                          
12307: 2016-02-23 17:32:52,888 INFO  libpurple: dnsquery:IP resolved for 149.154.175.100                                                                              
12307: 2016-02-23 17:32:52,888 INFO  libpurple: proxy:Attempting connection to 149.154.175.100                                                                        
12307: 2016-02-23 17:32:52,888 INFO  libpurple: proxy:Connecting to 149.154.175.100:443 with no proxy                                                                 
12307: 2016-02-23 17:32:52,888 INFO  libpurple: proxy:Connection in progress                                                                                          
12307: 2016-02-23 17:32:52,888 INFO  libpurple: dnsquery:IP resolved for 149.154.167.91                                                                               
12307: 2016-02-23 17:32:52,888 INFO  libpurple: proxy:Attempting connection to 149.154.167.91                                                                         
12307: 2016-02-23 17:32:52,888 INFO  libpurple: proxy:Connecting to 149.154.167.91:443 with no proxy                                                                  
12307: 2016-02-23 17:32:52,888 INFO  libpurple: proxy:Connection in progress                                                                                          
12307: 2016-02-23 17:32:52,889 INFO  libpurple: dnsquery:IP resolved for 149.154.171.5                                                                                
12307: 2016-02-23 17:32:52,889 INFO  libpurple: proxy:Attempting connection to 149.154.171.5                                                                          
12307: 2016-02-23 17:32:52,889 INFO  libpurple: proxy:Connecting to 149.154.171.5:443 with no proxy                                                                   
12307: 2016-02-23 17:32:52,890 INFO  libpurple: proxy:Connection in progress                                                                                          
12307: 2016-02-23 17:32:52,925 INFO  libpurple: proxy:Connecting to 149.154.167.51:443.                                                                               
12307: 2016-02-23 17:32:52,926 INFO  libpurple: proxy:Connected to 149.154.167.51:443.                                                                                
12307: 2016-02-23 17:32:52,928 INFO  libpurple: proxy:Connecting to 149.154.167.91:443.                                                                               
12307: 2016-02-23 17:32:52,928 INFO  libpurple: proxy:Connected to 149.154.167.91:443.                                                                                
12307: 2016-02-23 17:32:52,931 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #2 becomed ready                                                        
12307: 2016-02-23 17:32:52,932 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #4 becomed ready                                                        
12307: 2016-02-23 17:32:53,367 INFO  libpurple: proxy:Connecting to 149.154.175.50:443.                                                                               
12307: 2016-02-23 17:32:54,646 INFO  libpurple: proxy:Connected to 149.154.175.50:443.                                                                                
12307: 2016-02-23 17:32:54,649 INFO  libpurple: proxy:Connecting to 149.154.175.100:443.                                                                              
12307: 2016-02-23 17:32:54,649 INFO  libpurple: proxy:Connected to 149.154.175.100:443.                                                                               
12307: 2016-02-23 17:32:54,652 INFO  libpurple: proxy:Connecting to 149.154.171.5:443.                                                                                
12307: 2016-02-23 17:32:54,652 INFO  libpurple: proxy:Connected to 149.154.171.5:443.                                                                                 
12307: 2016-02-23 17:32:55,008 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #1 becomed ready                                                        
12307: 2016-02-23 17:32:55,009 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #3 becomed ready                                                        
12307: 2016-02-23 17:32:55,010 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #5 becomed ready

**Comment by majn** [comment]
Uhm, there is a debian repository with binaries?

**Comment by thundergreen** [comment]
sudo apt-get install telegram-purple <<< yes :) But it won't start here :(

**Comment by thundergreen** [comment]
SPECTRUM2 BACKEND LOG:

root@osmc:/var/log/spectrum2/telegram.emevth.no-ip.biz/backends# tail -f backend-1.log                                                                                
15657: 2016-02-23 18:07:04,138 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #4 becomed ready                                                        
15657: 2016-02-23 18:07:04,535 INFO  libpurple: proxy:Connecting to 149.154.175.50:443.                                                                               
15657: 2016-02-23 18:07:04,536 INFO  libpurple: proxy:Connected to 149.154.175.50:443.                                                                                
15657: 2016-02-23 18:07:04,538 INFO  libpurple: proxy:Connecting to 149.154.175.100:443.                                                                              
15657: 2016-02-23 18:07:04,538 INFO  libpurple: proxy:Connected to 149.154.175.100:443.                                                                               
15657: 2016-02-23 18:07:04,540 INFO  libpurple: proxy:Connecting to 149.154.171.5:443.                                                                                
15657: 2016-02-23 18:07:04,541 INFO  libpurple: proxy:Connected to 149.154.171.5:443.                                                                                 
15657: 2016-02-23 18:07:04,955 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #1 becomed ready                                                        
15657: 2016-02-23 18:07:07,312 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #3 becomed ready                                                        
15657: 2016-02-23 18:07:07,313 INFO  libpurple: prpl-telegram:outbound rpc connection from dc #5 becomed ready

**Comment by majn** [comment]
We have a running request for adding tgp to the debian repositories, but it hasnt been approved yet, which is why that line doesnt work (yet). I thought there may be a different person offering unofficial packages.

**Comment by BenWiederhake** [comment]
@thundergreen We strongly recommend building from source, since there should be no Debian archive yet. If you're using the ppa for Ubuntu, that's a bad idea too (waaay outdated version; also, Ubuntu and Debian packages are often enough incompatible.)
